hangons.js

```diff
@@ -198,6 +198,52 @@ function getName(id, participants)
     return id;
 }
 
+function imagify(inputText)
+{
+    var replacedText, replacePattern1, replacePattern2;
+
+    if (inputText === undefined)
+    {
+        inputText = "";
+    }
+
+		// Extract URL
```

Comment:
These comments seem to use tabs rather than the 4 space indentations of the rest of the file, which makes them look a little strange on the Github UI. It'd be best to keep consistency across the file.